CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
+
+Thanks for stopping by! This document should cover most topics surrounding contributing to this repo.
+
+* [How to contribute](#how-to-contribute)
+  * [Reporting issues](#reporting-issues)
+  * [Fixing issues](#fixing-issues)
+  * [Adding features](#adding-features)
+
+## Reporting issues
+Are you having trouble getting started? Please [contact us directly](mailto:support@bugsnag.com?subject=%5BGitHub%5D%20bugsnag-js-performance%20-%20having%20trouble%20getting%20started%20with%20BugSnag) for assistance with integrating BugSnag into your application.
+If you have spotted a problem with this module, feel free to open a [new issue](https://github.com/bugsnag/bugsnag-js-performance/issues/new?template=Bug_report.md). Here are a few things to check before doing so:
+
+* Are you using the latest version of BugSnag? If not, does updating to the latest version fix your issue?
+* Has somebody else [already reported](https://github.com/bugsnag/bugsnag-js-performance/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen) your issue? Feel free to add additional context to or check-in on an existing issue that matches your own.
+* Is your issue caused by this module? Only things related to `@bugsnag/browser-performance` should be reported here. For anything else, please [contact us directly](mailto:support@bugsnag.com) and we'd be happy to help you out.
```

Comment:
On JS we say:

> Only things related to `@bugsnag/js` (including `@bugsnag/plugin-*` plugins to support various frameworks)

@gingerbenw – can you suggest a suitable description for this repo's packages?

Reply:
Only things related to `@bugsnag/browser-performance`, `@bugsnag/react-native-performance`, and performance related plugins (such as `@bugsnag/plugin-react-navigation-performance`)

?